### PR TITLE
Survey/Corridor Scan note about survey area ground elevation

### PR DESCRIPTION
**Pull request by DonLakeFlyer**
@hamishwillee Can you take a look at this? I've run it by Todd Colten who has added his changes to what I wrote. So it's technically correct. But maybe not as good as it could from from a wording standpoint. Is it understandable?

### CODE REVIEW

File: en/toolbar/toolbar.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Main Toolbar
```
Comment (hamishwillee) [inline en/toolbar/toolbar.md:2]: This is good. Tempting to add an image of the whole toolbar here giving the icons context

File: en/PlanView/pattern_survey.md [hunk 2]
```diff
@@ -3,6 +3,14 @@
 A survey allows you to create a grid flight pattern over a polygonal area. 
 You can specify an arbitrary polygon, the angle and other properties of the grid, and camera settings appropriate for creating geotagged images.
 
+> **Important** If the survey area has significant elevation variation then consider enabling [Terrain Following](#terrain).
```
Comment (hamishwillee) [inline en/PlanView/pattern_survey.md:6]: @DonLakeFlyer I reworded with suggestion first, and link to the section that explains how you do this. The link is important, the restructure is to taste - I prefer to tell people the problem and solution first if possible to catch the lazy readers.
Upshot, if you want to revert, do so, but fix up the link.

File: en/PlanView/pattern_corridor_scan.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Corridor Scan (Plan Pattern)
 
-A corridor scan allows you to create a flight pattern that follows a poly-line. 
-This can be used to, for example, survey a road.
-It is supported on all autopilots and vehicle types.
+A corridor scan allows you to create a flight pattern that follows a poly-line. This can be used to, for example, survey a road. It is supported on all autopilots and vehicle types.
+
+> **Important** When planning a Corridor Scan using camera specifications the ground elevations under your survey area are assumed to be at the same altitude as your launch/home location. If the ground elevation under your survey area is either higher or lower than your launch/home location the effective overlap in your images will be less or more (respectively) than calculated. If ground elevation under your survey area is significantly higher than your launch/home location you could inadvertently plan a mission which causes the vehicle to fly into trees, obstacles, or the ground. Use Terrain Follow to create a survey that more closely maintains the desired altitude above terrain that has significant elevation differences from your launch/home altitude.
```
Comment (hamishwillee) [inline en/PlanView/pattern_corridor_scan.md:5]: See my comment https://github.com/mavlink/qgc-user-guide/pull/263/files#r423459117. You should link down to how they enable terrain following. Ideally you'd copy my pattern in section below, but thats up to you

**Comment by hamishwillee** [comment]
@DonLakeFlyer Sorry, missed this. This was fine, but I added some suggestions/changes that I think are slightly better.

**Comment by DonLakeFlyer** [comment]
Thanks @hamishwillee. These changes all seem fine

**Pull request merged**